README.md

```diff
@@ -57,6 +57,7 @@ conn_info = {'host': '127.0.0.1',
              # using server-side prepared statements is disabled by default
              'use_prepared_statements': False,
              # connection timeout is not enabled by default
+             # 5 seconds timeout for obtaining connection
```

Comment:
Thanks for polishing the readme! My concern is that "for obtaining connection" is not so accurate as of today, it is acutally socket timeout. Please see my comment at [here](https://github.com/vertica/vertica-python/issues/198#issuecomment-482299846).

Reply:
Thanks for the comment. I have updated the comment to  either TCP or socket operation